test/unit/visual/cases/typography.js

```diff
@@ -122,22 +124,25 @@ visualSuite("Typography", function () {
         { alignX: p5.LEFT, alignY: p5.BOTTOM },
         { alignX: p5.CENTER, alignY: p5.BOTTOM },
         { alignX: p5.RIGHT, alignY: p5.BOTTOM },
+        { alignX: p5.LEFT, alignY: p5.BASELINE },
+        { alignX: p5.CENTER, alignY: p5.BASELINE },
+        { alignX: p5.RIGHT, alignY: p5.BASELINE },
       ];
 
-      p5.createCanvas(300, 80);
-      p5.textSize(60);
+      p5.createCanvas(300, 200);
```

Comment:
For speed, each screenshot is resized to fit in 50x50 before comparison. Since this image (and the alignment test too) is so large compared to that, differences in antialiasing between devices could cause this test to fail.

You can call `screenshot()` multiple times per test and it will generate a separate image each time. Maybe we can make the canvas size smaller for these, and instead of placing each case somewhere else in one canvas, we can clear the canvas for each case and `screenshot()` them before moving on to the next?

Reply:
For added context, the test failure on CI right now is saying it expected this:
![image](https://github.com/user-attachments/assets/7c2c0475-d0e9-41a5-b317-68df3328238d)

but got this:
![image](https://github.com/user-attachments/assets/0d734ffe-4780-4c38-b42d-f7b83cf25295)